sample_factory/algo/utils/make_env.py

```diff
@@ -132,7 +132,7 @@ def step(self, actions: Tensor):
         ofs = 0
         next_ofs = self.single_env_agents
         for i, e in enumerate(self.envs):
-            idx = ofs if self.single_env_agents == 1 else slice(ofs, next_ofs)
+            idx = slice(ofs, next_ofs)
```

Comment:
Can you explain this change?
I think this breaks test_example.py 
If we have a single agent we want to reduce the dimensionality, i.e. go from [[action]] to [action]
If you use slice then you will still have [[action]] which will break some envs.

Unless I am missing something here

Reply:
The MultiAgentWrapper has another reduction in dimensionallity with 'action = action[0]' which was breaking mujoco since it changed a size-N action to size 1. If we want different behaviour with a single agent, maybe we should change the multi-agent wrapper instead?

Reply:
Ok, so this is not the right way to fix it because it breaks other envs.
I'm thinking now whether we even need MultiAgentWrapper if we're using SequentialVectorize. I think I should look at this problem tomorrow and develop a proper fix.

If you can give me instructions on how to reproduce this problem, it'd be great.

I hope this does not stop you from working on evaluations